modules/nf-core/fcsgx/fetchdb/main.nf

```diff
@@ -0,0 +1,46 @@
+process FCSGX_FETCHDB {
+    tag "$manifest.baseName"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ncbi-fcs-gx:0.5.4--h4ac6f70_1':
+        'biocontainers/ncbi-fcs-gx:0.5.4--h4ac6f70_1' }"
+
+    input:
+    val manifest // URL of manifest. Should not stage locally.
+
+    output:
+    path "$prefix"      , emit: database
+    path "versions.yml" , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    // def args = task.ext.args ?: ''
```

Comment:
I would still keep this in, even if it there are no more additional parameters - shouldn't do any harm and if they are added in the future then don't have to manually reactivate this anyway

Reply:
I commented out so the language server doesn't complain. I think it's a good idea to comment it out if it's not used.